linode/manager
# M3-5649: Add Engine to Database Details URL

## Description

- Adds a databases engine to the Cloud Manager URL
  - **Example:** `https://cloud.linode.com/databases/5` will now be `https://cloud.linode.com/databases/mysql/5`
- The motivation behind this change is that the API requires **both** `id` and `engine` to lookup the Databases's full details 
  - Credit to @codyfinn

## How to test

- Test all DBaaS routing
- Test all existing DBaaS features to ensure this change caused no breaking changes to the work we have done so far
